### PR TITLE
Added Lightbell18's Brazilian Portuguese Subtitles

### PR DESCRIPTION
**Pull request by himham-jak**

### CODE REVIEW

File: game/assets/jak1/text/game_text_ptbr.gs [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+(language-id 13) ;; this file is for brazilian portuguese ptbr
+(text-version jak1-v2)
+
+;; -----------------
+;; memcard overrides lol
+
+(#x0138
+  "POR FAVOR NÃO REMOVA OU INSIRA NENHUM PERIFÉRICO, DESLIGUE O SEU SISTEMA OU REMOVA O JOGO"
+  "POR FAVOR NÃO REMOVA OU INSIRA NENHUM PERIFÉRICO, DESLIGUE O SEU SISTEMA OU REMOVA O JOGO"
+  )
```
Comment (ManDude) [inline game/assets/jak1/text/game_text_ptbr.gs:11]: This file is only for 1 language, only specify one text per entry, not two.

**Review by ManDude — CHANGES_REQUESTED** [review]
Each

File: game/assets/jak1/subtitle/game_subtitle_ptbr.gd [hunk 2]
```diff
@@ -0,0 +1,2437 @@
...
+  (720 :offscreen "SAMOS" "EU PERGUNTEI AS PLANTAS, MAS ELAS NÃO SE LEMBRAM")
+  (818 :offscreen "SAMOS" "AS PLANTAS PERGUNTARAM AS PEDRAS, MAS AS PEDRAS NÃO SE RECORDAM")
+  (955 :offscreen "SAMOS" "ATÉ MESMO AS PEDRAS NÃO SE RECORDAM...")
+  (1064)
+  (1087 :offscreen "SAMOS" "CADA OSSO DO MEU CORPO ME DIZ QUE AS RESPOSTAS ESTÃO NAS COSTAS DE UM JOVEM GAROTO")
+  (1297 :offscreen "SAMOS" "ALHEIO AO SEU DESTINO, DESINTERESSADO NA BUSCA PELA VERDADE, E IGNORANDO MEUS ENSINAMENTOS!")
+  (1536)
+  (1542 :offscreen "SAMOS" "E POR QUE ELE DARIA OUVIDOS AO VELHO SAMOS?")
+  (1677)
+  (1683 :offscreen "SAMOS" "EU SOU APENAS O MESTRE DO ECO VERDE, UM DOS HOMENS MAIS SÁBIOS DO PLANETA"")
```
Comment (water111) [inline game/assets/jak1/subtitle/game_subtitle_ptbr.gd:28]: double quote at the end of this line

File: game/assets/jak1/text/game_text_ptbr.gs [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+(#x1000 "OPÇÕES DE CÂMERA"
+        "OPÇÕES DE CÂMERA")
+(#x1001 "NORMAL"
+        "NORMAL")
+(#x1002 "INVERTIDO"
+        "INVERTIDO")
+(#x1003 "CÂMERA EM 1A PESSOA HORIZONT."
+        "CÂMERA EM 1A PESSOA HORIZONT.")
+(#x1004 "CÂMERA EM 1A PESSOA VERTICAL"
+        "CÂMERA EM 1A PESSOA VERTICAL"")
```
Comment (water111) [inline game/assets/jak1/text/game_text_ptbr.gs:30]: double quote

File: game/assets/jak1/text/game_text_ptbr.gs [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+        "MINI-JOGO PESCARIA")
+(#x1089 "TEMA DE DESAFIO"
+        "TEMA DE DESAFIO")
+
+(#x1090 "ECO AZUL INFINITO"
+        "ECO AZUL INFINITO")
+(#x1091 "ECO VERMELHO INFINITO"
+        "ECO VERMELHO INFINITO")
+(#x1092 "ECO VERDE INFINITO"
+        "ECO VERDE INFINITO"")
```
Comment (water111) [inline game/assets/jak1/text/game_text_ptbr.gs:188]: quote

File: game/assets/jak1/text/game_text_ptbr.gs [hunk 5]
```diff
@@ -0,0 +1,346 @@
...
+(#x1093 "ECO AMARELO INFINITO""
```
Comment (water111) [inline game/assets/jak1/text/game_text_ptbr.gs:189]: quote

File: game/assets/jak1/subtitle/game_subtitle_ptbr.gd (hunk 2 above)
Comment (himham-jak) [reply game/assets/jak1/subtitle/game_subtitle_ptbr.gd:28]: fixed

File: game/assets/jak1/text/game_text_ptbr.gs (hunk 1 above)
Comment (himham-jak) [reply game/assets/jak1/text/game_text_ptbr.gs]: fixed

**Comment by himham-jak** [comment]
> you accidentally pushed some unrelated stuff btw

this should be fixed

File: game/assets/jak1/subtitle/game_subtitle_ptbr.gd [hunk 6]
```diff
@@ -0,0 +1,2437 @@
...
+("sksp0011" :hint #x240
+  (0 "DAXTER" "A GENTE DEVIA PERGUNTAR AO PESCADOR LÁ NO RIO DA SELVA PROIBIDA")
+  (136 "DAXTER" "SE NÓS PODEMOS PEGAR O BARCO DELE EMPRESTADO PRA IR NA ILHA DAS NÉVOAS!")
+  )
+
+("sksp0039" :hint #x25b
+  (0 "DAXTER" "EI, TEM MARCAÇÕES DE RELÂMPAGO AZUL NAQUELES POSTES")
+  (202 "DAXTER" "E TEM ECO AZUL SAINDO DAQUELA FONTE ALI.")
+  (391)
+ (400 "DAXTER" ESTÁ PENSANDO NO QUE EU ESTOU PENSANDO?")
```
Comment (Hat-Kid) [inline game/assets/jak1/subtitle/game_subtitle_ptbr.gd:937]: Missing quote

File: game/assets/jak1/subtitle/game_subtitle_ptbr.gd [hunk 7]
```diff
@@ -0,0 +1,2437 @@
...
+("MAI-AM01" :hint #x0 (0 "MAIA" "ACABE COM ELES DOIS DE UMA VEZ POR TODAS!"))
+("MAI-AM02" :hint #x0 (0 "MAIA" "ACABA COM ELES!"))
+("MAI-AM03" :hint #x0 (0 "MAIA" "HA HA HA!"))
+("MAI-AM04" :hint #x0 (0 "MAIA" "ECO NÃO VAI TE PROTEGER AGORA!"))
+("MAI-AM05" :hint #x0 (0 "MAIA" "FAÇA ALGUMA COISA!"))
+("MAI-AM06" :hint #x0 (0 "MAIA" "AGH!"))
+("MAI-AM07" :hint #x0 (0 "MAIA" "NÃOOOOO!"))
+("MAI-AM08" :hint #x0 (0 "MAIA" "VAMOS ABRIR O RESERVATÓRIO E DESTRUIR VOCÊS!"))
+("MAI-AM09" :hint #x0 (0 "MAIA" "ELES NÃO PODEM PEGÁ-LO!"))
+("sksp0394" :hint #x0 (0 "DAXTER" PEGA O ECO AMARELO!"))
```
Comment (Hat-Kid) [inline game/assets/jak1/subtitle/game_subtitle_ptbr.gd:2397]: Missing quote

File: game/assets/jak1/text/game_text_ptbr.gs [hunk 8]
```diff
@@ -0,0 +1,194 @@
...
+(#x1025 "POUPAR PERSONAGENS")
+(#x1026 "POUPAR FUNDO")
+(#x1027 "FORÇAR ENVIRONMENT MAPPING")
+
+(#x1030 "MOSTRAR JOGO NO DISCORD")
+(#x1031 "MODO DE EXIBIÇÃO")
+(#x1032 "MODO JANELA")
+(#x1033 "SEM BORDA")
+(#x1034 "TELA CHEIA")
+(#x1035 "RESOLUÇÃO DO JOGO)
```
Comment (ManDude) [inline game/assets/jak1/text/game_text_ptbr.gs:50]: Missing quote.

File: game/assets/jak1/text/game_text_ptbr.gs [hunk 9]
```diff
@@ -0,0 +1,194 @@
...
+(#x1116 "NORSK")
+
+
+;; -----------------
+;; test
+
+(#x7fff
+  "ARMADURA"
+  )
+
```
Comment (ManDude) [inline game/assets/jak1/text/game_text_ptbr.gs:193]: Do not translate.

**Review by Hat-Kid — CHANGES_REQUESTED** [review]

**Review by ManDude — CHANGES_REQUESTED** [review]
It looks like your `progress-pc.gc` is a bit messed up.

**Review by water111 — APPROVED** [review]

**Pull request merged**